src/utils.ts

```diff
@@ -362,14 +365,18 @@ export function extractInt(str: string): number {
 	return parseInt(str.match(/\d+/)![0]);
 }
 
-export function getFuzzySizeFactor(): number {
-	const { fuzzySizeThreshold } = getRuntimeConfig();
-	return fuzzySizeThreshold;
+export function getFuzzySizeFactor(searchee: Searchee): number {
```

Comment:
does this obviate fuzzySizeThreshold when you have seasonFromEpisodes turned on? is there a way to consolidate the options?

Reply:
Yes, `seasonFromEpisodes` completely replaces `fuzzySizeThreshold` for ensemble. `getFuzzySizeFactor(searchee)` and `getMinSizeRatio(searchee)` are used everywhere to account for this.

Reply:
Hmm, I haven't really given it much thought but if you think the config interface makes the most sense like this I trust you.